src/content/blog/whats-new-june-2024.mdx

```diff
@@ -0,0 +1,151 @@
+- 📺 [CURSO COMPLETO DE ASTRO](https://www.youtube.com/watch?v=wS4QN_xqwlM) by @informatica-live
+- 📺 [Astro JS - เฟรมเวิร์กน้องใหม่ เขียนง่าย โหลดไวมาก! by Chakkrisn Talawat](https://www.youtube.com/watch?v=WC5PRboDfJ0) by @creatorsgarden
+- 📺 [Sharing Session: Meet with Astro JS](https://www.youtube.com/watch?v=oEG5amJdMhQ) by @ngobrolintech
+- 📺 [DevTalles podcast - 167: Astro | Una opinión honesta y reflexiva](https://www.youtube.com/watch?v=WS49_hENfgE) by @DevTalles
+- 📺 [Capitole Tech Talks - View Transitions con Astro v4](https://www.youtube.com/watch?v=EnrsiLrGZ24) by @capitoleconsulting
+- 📺 [Astro Fixed Environment Variables!!](https://www.youtube.com/watch?v=kTceOl4qCm0) by @JamesQQuick
+- 📺 [My First Impressions (to Future of Astro)](https://www.youtube.com/watch?v=-NZIHSGJuzU) by @CodinginPublic
+- 📺 [Building LaunchFast w/Rishi Raj Jain](https://www.youtube.com/watch?v=ncFLhbiKFdU) by @brianmmdev
+- 📄 [Hybrid i18n with Next and Astro (part 1)](https://www.ericburel.tech/blog/hybrid-i18n-next-astro-1) by @ericbureltech
+- 📄 [Using React Query with Astro: A Step-by-Step Guide](https://hounie.me/writings/how-to-use-react-query-with-astro/) by @mihos8146
+- 📄 [How to Create a Persistent Sidebar Menu using Astro and View Transitions](https://jbhutch.com/blog/astro-with-peristent-sidebar/) by @ihiutch 
+- 📄 [Using Animate On Scroll (AOS) in Astro: A Step-by-Step Guide](https://www.launchfa.st/blog/aos-astro) by @rishirajjain
+- 📄 [Using Unplug Icons in Astro with React: A Step-by-Step Guide](https://www.launchfa.st/blog/react-astro-unplugin-icons) by @rishirajjain
+- 📄 [How to use Strapi with Astro](https://neon.tech/guides/strapi-cms) by @rishirajjain
+- 📄 [How to Build your own Reverse Image Search in Astro](https://neon.tech/guides/llamaindex-postgres-search-images) by @rishirajjain
+- 📄 [How to Build a RAG chatbot with Astro, Postgres, and LlamaIndex](https://neon.tech/guides/chatbot-astro-postgres-llamaindex) by @rishirajjain
+- 📄 [Building a Contact Form with Astro and Cloudflare Workers: A Simple Guide](https://hkbertoson.hashnode.dev/building-a-contact-form-with-astro-and-cloudflare-workers-a-simple-guide) by @hkbertoson
+- 📄 [Generate Related Posts for Your Astro Blog with Astra DB Vector Search](https://www.datastax.com/blog/generate-related-posts-for-your-astro-blog-with-astra-db-vector-search) by @datastax
+- 📄 [How to Implement RBAC (Role-Based Access Control) in Astro](https://www.permit.io/blog/how-to-implement-rbac-role-based-access-control-in-astro-framework) by @permit_io
+- 📄 [i like astro](https://www.cs.mun.ca/~jaharrhy/babble-bin/i-like-astro/) by @jackharrhy
+- 📄 [Adding search to static Astro sites](https://www.thomasledoux.be/blog/search-static-astro-website) by @thomasledoux1
+- 📄 [Integrate Pagefind's Search with Astro: A Complete Setup Guide](https://syntackle.com/blog/pagefind-search-in-astro-site/) by @murtuzaalisurti
+- 🎙️ [title]() by @
+- 🎙️ [title]() by @
+- 📺 [title]() by @
```

Comment:
Flagging that these will need deleting (although I guess that’s already on your radar)

```suggestion
```

Reply:
Yup! It's not the end of the month yet! I still need some placeholder elements. Thanks!